R/DataDistribution.R

```diff
@@ -24,14 +24,34 @@ setClass("DataDistribution", representation(
 )
 
 
-
-#' @rdname DataDistribution-class
+#' Probability density function
+#'
+#' \code{probability_density_function} evaluates the probability density
+#' function of a specific distribution \code{dist} at a point \code{x}.
+#' The mean is assumed to be \eqn{\sqrt{n} \theta}{n^(1/2)  theta}.
+#'
+#' @template dist
+#' @param x value where the pdf is evaluated
```

Comment:
repeated params should always be documented using @template for consistency (applies to all!)